index.html

```diff
@@ -195,12 +195,12 @@ <h3 class="article__title article__title--sidebar">
             <span class="fa fa-facebook-square"></span>
             All Facebook events
           </a>
-        </div>
+        </aside>
       </div>
       <!-- end of row -->
     </div>
 
-    <div class="footer container">
+    <footer class="footer container">
       <div class="row">
```

Comment:
As with `article` above you could change this just to target the `<footer>` tag without needing to define a footer class (i.e. change `.footer` to `footer` in the CSS).